dask/dataframe/core.py

```diff
@@ -578,6 +578,61 @@ def clear_divisions(self):
         divisions = (None,) * (self.npartitions + 1)
         return type(self)(self.dask, self._name, self._meta, divisions)
 
+    def calculate_divisions(self, col=None):
+        """Calculate the divisions for the dataframe.
+
+        This method triggers immediate computation. If you find yourself running this command
+        repeatedly for the same dataframe, we recommend storing the result
+        so if you re-run it, divisions won't be re-calculated.
```

Comment:
```suggestion
        so you don't have to rerun it.
        
        If the column or index is not already in sorted order, raises ``ValueError``.
```
Technically each partition itself doesn't have to be fully sorted, just that it's sorted _across_ the partitions / their mins and maxes don't overlap, but I can't think of how to explain that clearly, can you?

Reply:
Yeah, it seems like complexity that is probably not relevant to many. I am trying to think. I guess I can imagine reading in from daily files where each file is a chunk but the data within could be not sorted. Things would work for that scenario right? Maybe we can explain this with the word overlap? Like 
```suggestion
        so you don't have to rerun it.
        
        If the column or index values overlap between partitions, raises ``ValueError``.
        To prevent this, make sure the data are sorted by the column or index.
```

Reply:
^ yeah, I think this is good!